apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -1127,7 +1077,8 @@ export class WorkshopForm extends React.Component {
 
 export default connect(state => ({
   permission: state.workshopDashboard.permission,
-  facilitatorCourses: state.workshopDashboard.facilitatorCourses
+  facilitatorCourses: state.workshopDashboard.facilitatorCourses,
+  mapboxAccessToken: state.workshopDashboard.mapboxAccessToken
```

Comment:
I think since mapboxAccessToken is provided by redux, we don't need to pass it in as a prop from DetailsPanel